.circleci/config.yml

```diff
@@ -510,6 +510,43 @@ jobs:
           command: |
             docker push flowminder/<<parameters.component>>:$CIRCLE_SHA1
 
+  run_flowetl_tests:
+    docker:
+      - image: circleci/python:3.7
+      - image: flowminder/flowetl:$CIRCLE_SHA1
```

Comment:
Is this needed?

Reply:
I dunno? That's what you told me to do! I need to be able to `run` that image from this job...so do this or pull within the job?

Reply:
Iirc just the run command is fine - it’ll do the pull as well. I misunderstood with the original suggestion!

Reply:
Or save the image in workspace and load it? You don't have any other jobs that do stuff with an image outside the task that builds it as far as I can see?

Reply:
Ahh ok